README.md

```diff
@@ -46,10 +46,67 @@ The `veraison` command allows starting and stopping `veraison` services and
 viewing and manipulating Veraison logs and stores. See the output of `veraison
 -h` for the full list of available commands.
 
-> **Warning**: The docker deployment is not suitable for production. It is only
-> intended to be used in development environments. It is not hardened and
-> cannot handle high traffic volumes.
+> [!WARNING]
+> The docker deployment is not suitable for production. It is only intended to
+> be used in development environments. It is not hardened and cannot handle
+> high traffic volumes.
 
+### Creating a native deployment
+
+Clone and cd into the repo:
+
+    git clone https://github.com/veraison/services.git
+    cd services
+
+For a native deployment, there are additional dependencies to the ones listed
+in the "requirements" section above. We have bootstrap scripts for Ubuntu and
+Arch that ensure that the required dependencies are installed.
+
+    make bootstrap
+
+This will invoke the system package manager via sudo, so you may be prompted
+for your password.
+
+If you are on a different OS, please look at one of the scripts under
+deployments/native/bootstrap/ and install the equivalent packages for your
+system (on MacOS, they should be available via brew).
+
+Veraison will be deployed into the directory specified by `VERAISON_ROOT`
+environment variable (or into `${HOME}/veraison-deployment` if it is not set).
+
+    export VERAISON_ROOT=${HOME}/veraison-deployment
+
+You can build, deploy, and start Veraison services with the following sequence
+of commands:
+
+    make native-deploy
+
+The whole process might take a few minutes.
+
+You can interact with the deployment via the frontend,
+`${VERAISON_ROOT}/bin/varaison`. You can alias the script to just `veraison` by
+source an env file from the deployment:
+
+    source ${VERAISON_ROOT}/env/env.bash
+
+(there is an equivalent `env.zsh` for `.zsh`).
+
+If you are on a Linux distribution with systemd, Veraison services should be
+running as user systemd units. Otherwise, you can the services inside virtual
+terminals with
```

Comment:
```suggestion
If you are on a Linux distribution with systemd or on MacOSX, Veraison
services should be running as user systemd units. Otherwise, you can start
the services inside virtual terminals with
```